### ISSUE TITLE
Storybook 8 Support

### ISSUE THREAD

**Issue by JamesIves** [break-stuff/wc-storybook-helpers#47]
I've seen this [migration guide](https://storybook.js.org/docs/8.0/addons/addon-migration-guide#key-changes-for-addons) floating about for Storybook 8 support, not sure if there's any changes required for wc-storybook-helpers, but would be great to have support for it close after it launches!

**Comment by break-stuff** [comment]
Yeah, I think it's safe to start testing it now that it's in beta.

**Comment by gambolputty** [comment]
At the moment, Storybook 8 doesn't seem to work with the current `template` function as in:

```js
export const Default: Story = {
  render: (args) => template(args),
  args: {},
};
```

I haven't figured out yet which internals are breaking. At the moment I get this error in my preview:

```bash
TypeError: import_channels.default is not a constructor
    at mockChannel (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:8748:10)
    at AddonStore2.getChannel (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:8800:24)
    at useArgs (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:9170:24)
    at setArgObserver (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:12014:22)
    at syncControls (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:12004:3)
    at getTemplate (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:11913:3)
    at template (http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/wc-storybook-helpers.js?v=b0c76ae2:12061:31)
    at render (http://localhost:6006/src/components/bde-button/bde-button.stories.ts:39:20)
    at undecoratedStoryFn (http://localhost:6006/sb-preview/runtime.js:100:4790)
    at hookified (http://localhost:6006/sb-preview/runtime.js:84:10607)
```

**Comment by break-stuff** [comment]
It looks like the function they use to keep the controls in sync was changed in v8. Let me see if I can patch that.

**Comment by nkabrown** [comment]
I ran into the `import_channels.default is not a constructor` issue when migrating to Storybook v8 at work. I've raised a fix in PR #59.
